ci/appveyor/package.cfg

```diff
@@ -4,16 +4,21 @@ packages = {pkg_prefix}libsodium
 
 filename_stub = aeternity-windows
 
-display_name = aeternity for Windows
+display_name = Aeternity for Windows
 
-description = The full aeternity node for Windows.
+description = The full Aeternity node for Windows.
 
 version = 1.5.0-alpha.1
```

Comment:
I understand the plan is to consider this for the minerva branch. Could you please rebase on minerva branch?

Reply:
done